solr/solr-ref-guide/modules/configuration-guide/pages/solr-properties.adoc

```diff
@@ -68,6 +70,10 @@ NOTE: Properties marked with "!" indicate inverted meaning between old and new p
 
 |solr.logs.dir|solr.log.dir|server/logs|Specifies the directory where Solr logs are stored.
 
+|solr.logs.requestlog.retain.enabled|SOLR_REQUESTLOG_ENABLED|true|Specifies if request logs should be created.
```

Comment:
I insist that you say "Jetty" in here somehow so it's super clear this isn't logs of Solr, it's the Jetty webserver.  One could even argue the system property should reflect that, but I think it's fine.

Reply:
Now is the best time ever to rename the parameter...   have an idea?  I will update the ref guide..